Nativing/accounts/models.py

```diff
@@ -1,3 +1,87 @@
 from django.db import models
+from django.contrib.auth.models import PermissionsMixin, AbstractBaseUser, BaseUserManager
 
-# Create your models here.
+
+class CustomAccountManager(BaseUserManager):
+    def create_user(
+        self,
+        name, 
+        email, 
+        nickname,
+        date_of_birth,
+        user_image,
+        user_gender,
+        password=None
+    ):
+        user = self.model(
+            name=name,
+            email=email, 
+            nickname=nickname, 
+            date_of_birth=date_of_birth,
+            user_image=user_image,
+            user_gender=user_gender,
+        )
+        user.set_password(password)
+        user.is_superuser = False
+        user.save(using=self._db)
+        return user
+
+    def create_superuser(
+        self,
+        name, 
+        email, 
+        nickname, 
+        password
+    ):
+        user = self.model(
```

Comment:
```suggestion
        user = self.create_user(
```
`createsuperuser`로 생성한 계정이 일반 로그인도 안되는 것 같더라구요.
정확히 왜인지는 모르지만,, 
customuser 생성하는 메소드를 받아와서 이용하면 해결할 수 있습니다.
다만, null = true를 안해준 부분에 대해서는 임의로 데이터를 넣어주어야 superuser가 생성되어요

Reply:
훌륭합니다!